scripts/agent_monitor_training.py

```
@@ -0,0 +1,117 @@
+Example:
+    python scripts/agent_monitor_training.py ReceiptsTable-dc5be22 active_training.json
+"""
+
+from receipt_dynamo import DynamoClient
+import json
+import sys
+from typing import Dict, Any, Optional
+from datetime import datetime
+
+
+def monitor_active_training(table_name: str, output_file: str = None) -> Dict[str, Any]:
```

Ruff (0.14.5)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)